configure.ac

```diff
@@ -121,6 +121,11 @@ AC_ARG_ENABLE([s3],
                   [support Amazon AWS S3 URLs])],
   [], [enable_s3=check])
 
+AC_ARG_ENABLE([http-tests],
+  [AS_HELP_STRING([--disable-http-tests],
+                  [disables tests that require querying to a http server])],
+  [disable_http_tests=yes], [disable_http_tests=no])
```

Comment:
Also, it was decided that HTTP should be disabled by default. So the option should be renamed `--enable-http-tests` and the associated variable should only be set to true when this option is selected.